src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -167,7 +171,7 @@ interface Props {
   id: string;
   name: string;
   donations: DonationsType[];
-  status: string | undefined;
+  currentStatus: string | undefined;
```

Comment:
Would it make sense to switch this to the status enum instead of casting it on line 322?

```suggestion
  currentStatus: StatusEnum | undefined;
```